src/app/wallet/shared/signature-address-modal/signature-address-modal.component.ts

```diff
@@ -0,0 +1,155 @@
+import { Component, OnInit } from '@angular/core';
+import { AddressLookUpCopy } from '../../models/address-look-up-copy';
+import { AddressLookupComponent } from '../../addresslookup/addresslookup.component';
+import { MdDialog } from '@angular/material';
+import { RPCService } from '../../../core/rpc/rpc.service';
+import { Log } from 'ng2-logger';
+import { SignVerifyMessage } from '../../models/sign-verify-message';
```

Comment:
Organize this a bit more, the first imports are always the ones that we didn't create ourselves.
The second batch of imports are our own. Alphabetically is a nice touch :smile:

Reply:
@kewde its generic name, if you want to change tell me what it should ..? 
> SignVerifyMessage
>SignVerifyMessageForm etc ..